spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -6,15 +6,67 @@
 package com.datastrato.gravitino.spark.connector.hive;
 
 import com.datastrato.gravitino.spark.connector.PropertiesConverter;
+import com.google.common.collect.ImmutableMap;
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
+import java.util.Optional;
+import javax.ws.rs.NotSupportedException;
 
-/** Transform hive catalog properties between Spark and Gravitino. Will implement in another PR. */
+/** Transform hive catalog properties between Spark and Gravitino. */
 public class HivePropertiesConverter implements PropertiesConverter {
 
+  // Transform Spark format to Gravitino format
+  static final Map<String, String> hiveTableFormatMap =
+      ImmutableMap.of(
+          "sequencefile", "SEQUENCEFILE",
+          "rcfile", "RCFILE",
+          "orc", "ORC",
+          "parquet", "PARQUET",
+          "textfile", "TEXTFILE",
+          "avro", "AVRO");
```

Comment:
Are these properties just different in terms of case?

Reply:
the key is for spark which is not case sensitive, we convert to lower case

Reply:
In that case, can we just convert it to lowercase directly, without needing to use a map to handle it?

Reply:
If we just convert it to lowercase directly, we couldn't check the validity of the table format.